setup.py

```diff
@@ -1,10 +1,40 @@
 from setuptools import setup
 
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
 setup(
     name="zoop_wrapper",
     version="0.0.1",
-    description="",
-    url="https://github.com/imobanco/ZoopAPIWrapper",
     author="Imobanco",
-    packages=["zoop_wrapper"],
+    description="Cliente não oficial Zoop",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/imobanco/ZoopAPIWrapper",
+    packages=["zoop_wrapper", "zoop_wrapper.models"],
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
+        "Environment :: Web Environment",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Natural Language :: Portuguese (Brazilian)",
+        "Operating System :: OS Independent",
+        "Topic :: Documentation :: Sphinx",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+        "Topic :: Internet :: WWW/HTTP :: Dynamic Content :: CGI Tools/Libraries",
+        "Topic :: Utilities",
+        "",
+        "",
+    ],
+    python_requires=">=3.6",
+    install_requires=["requests>=2.23.0", "python-decouple>=3.3", "pycpfcnpj>=1.5.1",],
```

Comment:
[It is not considered best practice to use install_requires to pin dependencies to specific versions](https://packaging.python.org/discussions/install-requires-vs-requirements/)

Tem que ver como é o melhor jeito de fazer isso. Sei disso pq já tentei estudar/absorver isso, mas qual um bom jeito de fazer tem que pesquisar.

Reply:
Então, já estamos seguindo a instrução dos `lower_bounds`